improc/zogy.py

```diff
@@ -328,6 +375,10 @@ def zogy_add_weights_flags(ref_weight, new_weight, ref_flags, new_flags, ref_psf
         P_n /= np.sum(P_n)
         F_r = 1 / np.sqrt(np.sum(P_r ** 2) / B_r)
         F_n = 1 / np.sqrt(np.sum(P_n ** 2) / B_n)
-        D, P, S, Sc, al, al_err = zogy_subtract(R, N, P_r, P_n, np.sqrt(B_r), np.sqrt(B_n), F_r, F_n)
-        print(f'std(D)= {np.std(D)}, std(S)= {np.std(S)}, std(Sc)= {np.std(Sc)}')
+        output = zogy_subtract(R, N, P_r, P_n, np.sqrt(B_r), np.sqrt(B_n), F_r, F_n)
+        print(
```

Comment:
Yes, I think I accidentally left that in there. 
Usually when I am debugging I will use print() and remove or replace them with logger when I'm done.

Reply:
Ok, here this is just a testing ground under `if __name__ == "__main__":` so it should never actually be called. I'm leaving this as it is since this is only for debugging.